src/components/Footer.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { TodoStatus } from '../types/TodoStatus';
+
+type Props = {
+  status: TodoStatus,
+  setStatus: (value: TodoStatus) => void,
+  todos: Todo[],
+  clearCompletedTodos: () => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  status,
+  setStatus,
+  todos,
+  clearCompletedTodos,
+}) => {
+  const activeTodoCount = todos.filter(todo => !todo.completed).length;
+
+  const completedTodoCount = todos.filter(todo => todo.completed).length;
+
+  const handlerStatusAll = () => {
```

Comment:
Typo in the function name. Consider renaming 'handlerStatusAll' to 'handleStatusAll'.